packages/client/hmi-client/src/components/widgets/tera-input-field.vue

```diff
@@ -1,52 +1,28 @@
 <template>
-	<div
-		class="input-container"
-		:class="{ error: errorMessage, disabled: disabled, focused: focused }"
-		@click="focusInput"
-	>
-		<label for="custom-input" class="input-label">{{ label }}</label>
-		<input
-			id="custom-input"
-			class="input-field"
-			ref="inputField"
-			:value="modelValue"
-			@input="updateValue"
-			:disabled="disabled"
-			@focus="onFocus"
-			@blur="onBlur"
-		/>
+	<div class="input-container" :class="{ error: errorMessage }" @click.self.stop="focusInput">
```

Comment:
You do not need to use a class here. You can taget `div` directly as the style is scoped.
```suggestion
	<div :class="{ error: errorMessage }" @click.self.stop="focusInput">
```

Reply:
done